pendulum_msgs/msg/JointState.msg

```diff
@@ -0,0 +1,3 @@
+float64 position
```

Comment:
It's fine to use this for demo purposes. What I'd like to see in the long run is the ability to preallocate messages in publishers and subscribers, so dynamically-sized fields can be used. Something like:
```c++
sensor_msgs::msg::JointState msg;
msg.name.push_back("pendulum");
msg.position.resize(1, 0.0);
msg.velocity.resize(1, 0.0);
msg.effort.resize(1, 0.0);

publisher.preallocate(msg);  // optional
subscriber.preallocate(msg); //
```
`preallocate` would initialize with `msg` all internal buffers in the non real-time path. When publishing and subscribing from the real-time path, no further allocations are required as long as the vector and string sizes remain the same.

I don't know if there are details on how DDS works that would add additional complexity to this argument.